plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -217,13 +217,30 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
 
     override fun embedFunctionSignature(symbol: FirFunctionSymbol<*>): FunctionSignature {
         val retType = embedType(symbol.resolvedReturnTypeRef.coneType)
-        val receiverType = symbol.receiverType
-        val isReceiverUnique = symbol.receiverParameter?.isUnique(session) ?: false
-        val isReceiverBorrowed = symbol.receiverParameter?.isBorrowed(session) ?: false
+        val dispatchReceiverType = symbol.receiverType
+        val extensionReceiverType = symbol.extensionReceiverType
+        val isExtensionReceiverUnique = symbol.receiverParameter?.isUnique(session) ?: false
+        val isExtensionReceiverBorrowed = symbol.receiverParameter?.isBorrowed(session) ?: false
```

Comment:
Is this information always about the extension receiver?  That seems a bit strange

Reply:
At least I don't quite understand here what would be the syntax for dispatch. If it is not possible to mark the dispatch receiver as unique directly, I suppose we could introduce a special annotation `@MemberUnique` for member functions specifically.

Reply:
@francescoo22 We definitely want to be able to have the dispatch receiver be unique.  I would expect `@receiver:Unique` to work.

Reply:
As far as I know it's not possible to annotate the dispatch receiver with `@receiver:Unique`.
I think @GrigoriiSolnyshkin is right, we need to introduce a special annotation like `@receiverUnique` and annotate the function.

This is what the Kotlin doc says about the receiver target for annotations: "`receiver` (receiver parameter of an extension function or property)"

Reply:
Ah, that's annoying.  LGTM then